config/samples/hstreamdb_complete_sample.yaml

```diff
@@ -27,6 +27,33 @@ spec:
       targetPort: server-port
       nodePort: 31777
 ---
+# # Ref: https://github.com/hstreamdb/hstream/blob/main/conf/hstream.yaml
```

Comment:
Main changes here.

Reply:
Actually, the configMap should contain everything in the origin config file, not just the hserver part. So, It seems more appropriate to put this part in a separate file

Reply:
After discussions elsewhere, we are now supporting partial config here.